src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy

```diff
@@ -373,21 +373,15 @@ class ProtectedPathSpec extends HealthCheckSpecification {
     @Tags([ISL_RECOVER_ON_FAIL, ISL_PROPS_DB_RESET])
     def "Flow swaps to protected path when main path gets broken, becomes DEGRADED if protected path is unable to reroute(no bw)"() {
         given: "Two switches with 2 diverse paths at least"
-        //def switchPair = switchPairs.all().withAtLeastNNonOverlappingPaths(2).random()
-        //https://github.com/telstra/open-kilda/issues/5608
-        def switchesWhere5608IsReproducible = topology.activeSwitches.findAll {it.dpId.toString().endsWith("08")
-        ||it.dpId.toString().endsWith("09")}
-        def switchPair = switchPairs.all()
-                .excludeSwitches(switchesWhere5608IsReproducible)
-                .withAtLeastNNonOverlappingPaths(2).random()
+        def switchPair = switchPairs.all().withAtLeastNNonOverlappingPaths(2).random()
```

Comment:
What exactly is the fix? https://github.com/telstra/open-kilda/issues/5608 is still open and not solved, as far as I can see.

Reply:
Yes, let me add some details to explain my PR.

So, I was debugging a couple of tests (namely, `"Flow swaps to protected path when main path gets broken, becomes DEGRADED if protected path is unable to reroute(no bw)"` and `"Flow swaps to protected path when main path gets broken, becomes DEGRADED if protected path is unable to reroute(no path)"` because they failed sometimes for the past month. The failure was related to the same `Couldn't find non overlapping protected path. Skipped creating it` error message mentioned in #5608. So, even with the WA to skip 8-9 switches, the overlapping issue was still reproducible.

* `Flow swaps to protected path when main path gets broken, becomes DEGRADED if protected path is unable to reroute(no bw)` - here the issue was still reproducible between the switches 2-3, please see the screenshot below.
![image](https://github.com/telstra/open-kilda/assets/11521041/d86b8415-b67c-4654-9dd5-e2d70f15039b)
When I investigated this test, it turned out that the protected path ISL has enough bw, that is why the test cannot get expected "Not enough bandwidth or no path found" message.  So the fix in this PR is done to reduce the BW on the original protected path ISLs. So, with this fix, the test passes because the flow is degraded due to the expected "Not enough bandwidth or no path found" message. So, now no need to skip 8-9 switches since with the correct BW reducing, the test passes.
* `Flow swaps to protected path when main path gets broken, becomes DEGRADED if protected path is unable to reroute(no path)` - this test also failed sometimes due to Overlapping issue - even with the WA to skip 8-9 switches. I made the PR #5645 to fix this test because the ISLs were not broken correctly there. But now I see that with this fix from #5645, the WA to skip 8-9 can be removed, it is not reproduced anymore.

So, I have executed both these tests 20 times with just 8-9 switchpair, 2-3 switchpair, random switchpair and the overlapping issue is not reproduced anymore - the tests are passing. Thus, there is no need to exclude 8-9 switches from the switchpair anymore.